ext/phar/util.c

```diff
@@ -1579,7 +1579,13 @@ int phar_verify_signature(php_stream *fp, size_t end_of_phar, uint32_t sig_type,
 			}
 
 			md_ctx = EVP_MD_CTX_create();
-			EVP_VerifyInit(md_ctx, mdtype);
+			if (!md_ctx || !EVP_VerifyInit(md_ctx, mdtype)) {
+				EVP_MD_CTX_destroy(md_ctx);
+				if (error) {
+					spprintf(error, 0, "openssl signature could not be processed");
```

Comment:
```suggestion
					spprintf(error, 0, "openssl signature could not be verified");
```

Reply:
Thanks for reviewing. I used the same wording as below. Should those messages be changed from processed to verified as well?

Reply:
Probably not worth to change others - there is actually one with verified and this one is for VerifyInit so might make sense to just not introduce another processed. Doesn't really matter that much though as this warning is very unlikely to ever happen.